runtime/lua/vim/filetype.lua

```diff
@@ -23,36 +31,44 @@ local function starsetf(ft, opts)
 end
 
 ---@private
---- Get a single line or line range from the buffer.
---- If only start_lnum is specified, return a single line as a string.
---- If both start_lnum and end_lnum are omitted, return all lines from the buffer.
----
+--- Get a single line from the buffer.
+---@param bufnr integer The buffer to get the lines from
+---@param start_lnum integer The line number of the first line (inclusive, 1-based)
+---@return string
+function M.getline(bufnr, start_lnum)
+  -- Return a single line
+  return api.nvim_buf_get_lines(bufnr, start_lnum - 1, start_lnum, false)[1] or ''
```

Comment:
seems very special-purpose. and it's public?

Reply:
It's exposed but marked private.

Reply:
can we use underscore prefix for these kinds of cases, so they are easy to omit from `:lua vim.<tab>` (future enhancement)

Reply:
done